stock_move_location/init_hook.py

```diff
@@ -1,9 +1,5 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
-from odoo import SUPERUSER_ID, api
-
-
-def enable_multi_locations(cr, registry):
```

Comment:
Are you sure about this removal ? Default user is admin, not OdooBot

Reply:
I'm sure about it